### ISSUE TITLE
Load configuration not working

### ISSUE THREAD

**Issue by VRusnac** [sqlfluff/vscode-sqlfluff#81]
Hi guys,

sqlfluff extension doesn't load config. Here are the specifications:
``` 
{
    "sqlfluff.executablePath": "${workspaceFolder}/.venv/bin/sqlfluff",
    "sqlfluff.config": "${workspaceFolder}/pyproject.toml",
    "sqlfluff.linter.run": "onSave",
}
```
This is currently my `settings.json` in the workspace. When I save the SQL file it doesn't show any errors. If I comment config line it works fine. I tried to specify exact PATH to the config but it didn't work.

If I run command line it works fine:
`sqlfluff lint --config $PWD/pyproject.toml sample.sql `

My specs are:
- Python 3.9.6
- sqlfluff 1.4.4
- vscode extension sqlfluff v2.2.1

**Comment by RobertOstermann** [comment]
@VRusnac Currently the config setting does not allow for variable input, so an exact path would be necessary. I will add the option for variables in the next sqlfluff version, which should be released later today. Could you check the output channel "sqlfluff" and let me know what error you are seeing?

**Comment by VRusnac** [comment]
@RobertOstermann Sorry for delayed reply. I was on vacation. I've restarted my PC and reinitialised my python env and it worked nicely. Thank you !
